ament_pclint/package.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>ament_pclint</name>
+  <version>0.0.3</version>
+  <description>
+    The ability to perform static code analysis on C/C++ code using pclint
+    and generate xUnit test result files.
+  </description>
+  <maintainer email="jp.samper@apex.ai">Juan Pablo Samper</maintainer>
+  <license>Apex.AI Proprietary License</license>
+
+  <exec_depend>pclp64</exec_depend>
```

Comment:
For the release this needs to be a valid `rosdep` key. If that is not feasible the dependency needs to be removed.

Reply:
My guess is this won't be the case since it's a proprietary license, so I'll remove.